packages/libtool.rb

```diff
@@ -8,22 +8,22 @@ class Libtool < Package
   source_sha256 'e3bd4d5d3d025a36c21dd6af7ea818a2afcd4dfc1ea5a17b39d7854bcd0c06e3'
```

Comment:
Please increment the version:
`version '2.4.6-4'`

Reply:
Corrected in commit 495c345